File: android/src/main/java/com/getcapacitor/community/classes/events/UserIDEvent.java
package com.getcapacitor.community.classes.events;

import androidx.annotation.NonNull;
import com.getcapacitor.JSObject;
import java.util.UUID;

public class UserIDEvent {

    @NonNull
    UUID userID;

    public UserIDEvent(@NonNull UUID userID) {
        this.userID = userID;
    }

    @NonNull
    public JSObject toJSObject() {
        JSObject result = new JSObject();

        result.put("userID", userID.toString());

        return result;
    }
}

File: ios/Sources/BridgefyPlugin/Classes/Events/UserIDEvent.swift
import Foundation
import Capacitor

@objc public class UserIDEvent: NSObject {
    let userID: UUID

    init(_ userID: UUID) {
        self.userID = userID
    }

    public func toJSObject() -> JSObject {
        var result = JSObject()

        result["userID"] = userID.uuidString.lowercased()

        return result
    }
}
